src/utils/twoFactor.js

```diff
@@ -2,29 +2,40 @@ import * as nearApiJs from 'near-api-js'
 import { store } from '..'
 import { WalletError } from './walletError'
 import { promptTwoFactor } from '../actions/account'
-import { ACCESS_KEY_FUNDING_AMOUNT, convertPKForContract, toPK } from './wallet'
+import { convertPKForContract, toPK } from './wallet'
 
-const { transactions: {
-    deleteKey, addKey, functionCall, functionCallAccessKey, deployContract
-}} = nearApiJs
+const { 
+    Account,
+    transactions: { deleteKey, addKey, functionCall, functionCallAccessKey, deployContract }
+} = nearApiJs
 export const METHOD_NAMES_LAK = ['add_request', 'add_request_and_confirm', 'delete_request', 'confirm']
 const VIEW_METHODS = ['get_request_nonce', 'list_request_ids']
 const METHOD_NAMES_CONFIRM = ['confirm']
-const LAK_ALLOWANCE = process.env.LAK_ALLOWANCE || '10000000000000'
-const actionTypes = {
-    'functionCall': 'FunctionCall'
-}
+const GAS_2FA = process.env.REACT_APP_GAS_2FA || '100000000000000'
+const ALLOWANCE_2FA = process.env.ALLOWANCE_2FA || nearApiJs.utils.format.parseNearAmount('1')
```

Comment:
let's use `WALLET_2FA_ALLOWANCE` and `WALLET_2FA_GAS` environment variables

Reply:
ok